Site/CuratedContentEditorialWorkflow.md

```diff
@@ -53,18 +53,20 @@ each of the numbered steps in the description below.
      count.
    * Any more than 30 days old that have *not* garnered sufficient interest are labeled
      as `no interest` and closed.
-   * Any issues with sufficient interest are moved to *In Progress*. An *In Progress* issue
-     must have an **Author** and an **EB Member** who is different from the **Author**.
-     The **EB Member** will be the assignee of the issue. The **Author** will be indicated
+   * Any issues with sufficient interest are either moved to *Ready for Assignment* or *Ready to Write*. 
+     An issue is moved to *Ready to Assignment* in case there is no author or no EB member available for that 
```

Comment:
Typo: Ready **to** Assignment --> Read **for** Assignment

Reply:
Thanks